cob_cartesian_controller/src/trajectory_profile_generator/trajectory_profile_generator_builder.cpp

```diff
@@ -0,0 +1,55 @@
+
+/*!
+ *****************************************************************
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2015 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_twist_controller
+ *
+ * \author
+ *   Author: Christoph Mark, email: christoph.mark@ipa.fraunhofer.de / christoph.mark@gmail.com
+ *
+ * \date Date of creation: June, 2015
+ *
+ * \brief
+ *   This module contains the implementation of all interface types.
+ *
+ ****************************************************************/
+#define RAMP_PROFILE 1
+#define SINOID_PROFILE 2
```

Comment:
please remove the define makros and use the defines from the messages directly https://github.com/ipa-fxm/cob_control/blob/test_new_cartesian_controller/cob_cartesian_controller/msg/Profile.msg#L1-L2